src/api/Visualization/index.ts

```diff
@@ -76,6 +77,29 @@ export class VisualizationAPI extends Request {
     });
   }
 
+  /**
+   * Release the exclusive edit lock on a visualization.
+   */
+  public releaseEditLock(params: ReleaseVisualizationEditLockParams) {
```

Comment:
@hsenbassam This method is async, it returns a promise. As outlined by @Diachk, the call to `sendBeacon` has most likely to be done synchronously to guarantee that the request is scheduled, so it should be in an async function.

Let's not merge this pull request too quickly. We should first implements the frontend logic and carefully test that viz locks are released even if ones' browser is abruptly closed.

ping @aminerhanemi fyi

Reply:
Thank you @gmasclet for the info, I will based my frontend based on this rest-client and the one related in the server, we will not merge anything until we make sure everything is okay.